core/arch/asm_defines.asm

```diff
@@ -134,6 +134,8 @@
 # define DECLARE_FUNC_SEH(symbol) DECLARE_FUNC(symbol)
 # define PUSH_SEH(reg) push reg
 # define PUSH_NONCALLEE_SEH(reg) push reg
+# define ADD_STACK_ALIGNMENT sub REG_XSP, FRAME_ALIGNMENT - ARG_SZ
```

Comment:
Seems cleanest to move the #define of FRAME_ALIGNMENT, which is currently down on line 531, up above this use.  The comment there seems stale: there is no define in arch_exports.h.  The only other one I see is in x86.asm: but couldn't it get it from here?  If so that x86.asm comment should be moved here.

Reply:
Done.